.github/workflows/jekyll.yml

```diff
@@ -75,8 +75,14 @@ jobs:
 
       - run: pip install linkchecker --user
 
-      - name: Link Checker
-        run: |
-          linkchecker https://duckdb.org/ --config .github/linkchecker/linkchecker.conf --file-output html/utf-8/report.html
-      - run: cat report.html >> $GITHUB_STEP_SUMMARY
+      - name: Link Checker on latest
+        run: linkchecker https://duckdb.org/ --ignore-url 'https://duckdb\.org/docs/archive/.*' --config .github/linkchecker/linkchecker.conf --file-output html/utf-8/report.html
+
+      - run: cat report.html >> ${GITHUB_STEP_SUMMARY}
+        if: always()
+
+      - name: Link Checker on everything
+        run: linkchecker https://duckdb.org/ --config .github/linkchecker/linkchecker.conf --file-output html/utf-8/report.html
+
+      - run: cat report.html >> ${GITHUB_STEP_SUMMARY}
```

Comment:
Are we wanting to override the report? Or append to it?

Reply:
It would make sense to override. The second step runs a superset of the first step, so if the CI gets to it, we'll only need the output of the second step.